.gitignore

```diff
@@ -10,14 +10,15 @@
 ._.DS_Store
 .htpasswd
 .pidfile
-npm-debug.log
 .digest-cache.json
 check.txt
 .vagrant
 *.heapsnapshot
 
-# YARN local install
+# package managers
 .yarn
+npm-debug.log
+package-lock.json
```

Comment:
If we consider using them eventually... in most locking systems I have seen (mostly elixir and go vendor files), you would def. want to put them in your repo, hence my confusion.